tests/unit/mr.c

```diff
@@ -67,10 +67,9 @@ int main(int argc, char *argv[])
 	const size_t cache_init_size = 16;
 
 	/* Doesn't have to be correct -- for functionality test only */
-	const size_t system_page_size = 1024;
+	const size_t fake_page_size = 1024;
```

Comment:
😄

Reply:
it's about preventing shadowing with our global `system_page_size`.